packrat/packrat.opts

```diff
@@ -0,0 +1,19 @@
+auto.snapshot: FALSE
```

Comment:
@xilinchen I've left this file in the git history since it looks like these are configuration options we can set for the repo if we are not satisfied with the default options.